run_unittests.py

```diff
@@ -6036,7 +6037,7 @@ def test_introspect_installed(self):
             }
             the_truth_2 = {'/usr/lib/libsome.dylib',
                            '/usr/lib/libsome.0.dylib',
-            }
+                           }
```

Comment:
nitpick: Maybe this block should match the indentation of `the_truth`?

Reply:
`flake8` with default settings (not overridden by me) complained about it--maybe it was a false positive?

Reply:
I'm saying it should be:

```py
the_truth_2 = {
    '/usr/lib/libsome.dylib',
    '/usr/lib/libsome.0.dylib',
}
```

Like `the_truth` above it. Flake8 should accept this.

Reply:
For me, flake8 says `E124 closing bracket does not match visual indentation` but it does match project style, so I put it as you mentioned, thanks